packages/core/src/date/date.stories.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/core/date/register.js';
+import { html } from 'lit-html';
+import { getElementStorybookArgs, spreadProps, getElementStorybookArgTypes } from '@clr/core/internal';
+import customElements from '../../dist/core/custom-elements.json';
+
+export default {
+  title: 'Forms (Preview)/Date (Internal)/Stories',
+  component: 'cds-date',
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=20%3A1',
```

Comment:
should this point to datepicker in figma?

Reply:
fixed